src/linagora.esn.james/app/quota/user/james-quota-user.controller.spec.js

```diff
@@ -13,14 +13,16 @@ describe('The jamesQuotaUserController', function() {
   var user = { _id: 'userId' };
 
   beforeEach(function() {
-    module('linagora.esn.james');
-    module('esn.configuration', function($provide) {
+    angular.mock.module('linagora.esn.james');
+    angular.mock.module('esn.configuration', function($provide) {
+      const $q = angular.injector(["ng"]).get("$q");
```

Comment:
Can we avoid defining this in multiple places by having a `before` or `beforeEach` block?

Reply:
Fixed